app/models/apple/episode.rb

```diff
@@ -384,16 +384,20 @@ def remove_episode_audio_container_parameters
     end
 
     def publishing_state_bridge_params(state)
+      self.class.publishing_state_bridge_params(apple_id, state)
+    end
+
+    def self.publishing_state_bridge_params(apple_id, state)
       {
         request_metadata: {
           apple_episode_id: apple_id
         },
-        api_url: api.join_url("episodePublishingRequests").to_s,
-        api_parameters: publishing_state_parameters(state)
+        api_url: Apple::Api.join_url("episodePublishingRequests").to_s,
+        api_parameters: publishing_state_params(apple_id, state)
```

Comment:
yeah, that makes sense, pass through the apple_id instead of from the instance state/methods.